### PR TITLE
Move tf provider2

### PR DESCRIPTION
**Pull request by mandoonandy**
example.go is moved to client-go/examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,14 @@ Deployment choices
 
 libvirt is the only supported hypervisor. Instances are specified to libvirt with simple templated XML. If your local requirements are different to what's in the template, you're welcome to change the template to meet your needs. If you're template changes break things, you're also welcome to debug what went wrong for yourself.
 
+Tools to control ShakenFist
+===========================
+Golang API Client Library   https://github.com/shakenfist/client-go
+
+Terraform Provider  https://github.com/shakenfist/terraform-provider-shakenfist
+
+Python Client  https://github.com/shakenfist/client-python
```
Comment (mikalstill) [inline README.md:27]: We should skip this one for now until we actually move things.

File: README.md [hunk 2]
```diff
@@ -18,6 +18,14 @@ Deployment choices
 
 libvirt is the only supported hypervisor. Instances are specified to libvirt with simple templated XML. If your local requirements are different to what's in the template, you're welcome to change the template to meet your needs. If you're template changes break things, you're also welcome to debug what went wrong for yourself.
 
+Tools to control ShakenFist
```
Comment (mikalstill) [inline README.md:21]: The name is "Shaken Fist" for better or worse...

File: README.md [hunk 3]
```diff
@@ -18,6 +18,14 @@ Deployment choices
...
+===========================
+Golang API Client Library   https://github.com/shakenfist/client-go
+
+Terraform Provider  https://github.com/shakenfist/terraform-provider-shakenfist
```
Comment (mikalstill) [inline README.md:25]: Do we think these should be a bulleted list like:

* Goland API Client Library: https://github.com/shakenfist/client-go
* Terraform Provider: https://github.com/shakenfist/terraform-provider-shakenfist

File: README.md (hunk 1 above)
Comment (mandoonandy) [reply README.md:27]: Agree. I "was" planning to submit all changes at the same time but I'm fighting golang package import issues...

File: README.md (hunk 3 above)
Comment (mandoonandy) [reply README.md:25]: yep. neater.

File: README.md (hunk 1 above)
Comment (mandoonandy) [reply README.md:27]: Removing until other pull requests processed

**Comment by mandoonandy** [comment]
Merge the deletions. I'll circle back for the readme after everything has been moved

**Comment by mikalstill** [comment]
That's fair. I think the README needs a general refactor anyway (probably to something shorter and then a set of things in a docs directory).

**Pull request merged**